package-lock.json

```diff
@@ -16,6 +16,7 @@
         "@rdfjs-elements/rdf-snippet": "^0.4.3",
         "@rdfjs/term-set": "^2.0.1",
         "@vaadin/vaadin": "^22",
+        "@vaadin/vaadin-lumo-styles": "^1.6.1",
```

Comment:
This is wrong. The packages already includes lumo-styles v22, matching the version of `@vaadin/vaadin`. Please revert

Reply:
I reverted this